src/Files.Uwp/Views/ColumnShellPage.xaml.cs

```diff
@@ -172,7 +177,14 @@ public ColumnShellPage()
             ToolbarViewModel.ToolbarPathItemLoaded += ColumnShellPage_ToolbarPathItemLoaded;
             ToolbarViewModel.AddressBarTextEntered += ColumnShellPage_AddressBarTextEntered;
             ToolbarViewModel.PathBoxItemDropped += ColumnShellPage_PathBoxItemDropped;
-            ToolbarViewModel.BackRequested += ColumnShellPage_BackNavRequested;
+
+            /*
+              
+            TODO UA307 Default back button in the title bar does not exist in WinUI3 apps.
+            The tool has generated a custom back button in the MainWindow.xaml.cs file.
+            Feel free to edit its position, behavior and use the custom back button instead.
+            Read: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-1#restoring-back-button-functionality
+            */ToolbarViewModel.BackRequested += ColumnShellPage_BackNavRequested;
```

Comment:
Wasn't this our own implementation, I think we can revert this.

Reply:
Lol the converter added this automatically. The problem is were this was commented (support for system back button): `SystemNavigationManager.GetForCurrentView().BackRequested`